CHANGELOG/CHANGELOG-1.0.md

```diff
@@ -0,0 +1,12 @@
+# First Release 
```

Comment:
Can you put the release version there?

Do you want the first release to be 1.0? or 0.1?

Reply:
We should probably do a 0.1 to start with.